docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,407 @@
+| Set `params.uid2Token` to the response body as a JavaScript object. | See [Client Refresh uid2Token Example](#client-refresh-uid2token-example). | You might choose to provide the response body via `params.uid2Token` in either of these cases:<ul><li>If you are already storing a lot of data in the cookie and adding the response body might exceed the cookie size limit.</li><li>If you prefer to have the the Prebid module store the token value for you.</li></ul> |
+
+#### Client Refresh Mode Cookie Example
+
+In Client Refresh mode, Prebid.js takes care of refreshing the token. To do this, you must configure it to store the token. The following example shows the cookie and also the configuration for storing the token in a cookie called `uid2_pub_cookie`.
+
+Cookie:
+
+```
+uid2_pub_cookie={"advertising_token":"...advertising token...","refresh_token":"...refresh token...","identity_expires":1684741472161,"refresh_from":1684741425653,"refresh_expires":1684784643668,"refresh_response_key":"...response key..."}
+```
+
+Configuration:
+
+```js
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        uid2Cookie: 'uid2_pub_cookie'
+      }
+    }]
+  }
+});
+```
+
+For an example of the token, see [Sample Token](#sample-token).
+
+#### Client Refresh Mode uid2Token Example
+
+The following example shows a sample configuration. For the contents of the token, see [Sample Token](#sample-token).
+
+```javascript
+pbjs.setConfig({
+  userSync: {
+    userIds: [{
+      name: 'uid2',
+      params: {
+        uid2Token: {
+          'advertising_token': '...advertising token...',
+          'refresh_token': '...refresh token...',
+          // etc. - see the sample token for contents of this object
+        }
+      }
+    }]
+  }
+});
+```
+
+#### Passing a New Token: Client Refresh Mode
+
+If the refresh token expires, you'll need to supply a new token response so that a new advertising token and a new refresh token are available for future refreshes. 
+
+For information on how to determine if you need to provide a new token, see [Determining Whether the Module Has a Valid Token](#determining-whether-the-module-has-a-valid-token).
+
+### Server-Only Mode
+
+In server-only mode, only the advertising token is provided to the module. The module cannot refresh the token. You are responsible for implementing a way to refresh the token.
+
+To configure the module to use server-only mode, do **one** of the following:
+
+| Implementation Method | Link to Example |
+| --- | --- |
+| Set a cookie named `__uid2_advertising_token` and store the advertising token value in it. | [Server-Only Cookie Example](#server-only-cookie-example) |
+| Set `value` to an ID block containing the advertising token. | [Server-Only Value Example](#server-only-value-example) |
```

Comment:
the 2 links on 2 lines above are broken. they should be 
#server-only-mode-cookie-example
#server-only-mode-value-example)

Reply:
@sunnywu TYSM. Fixed. Checked for other instances, for both -- there are none.